src/xbiscuit.c

```diff
@@ -0,0 +1,285 @@
+#include "common.h"
+#include "libdma.h"
+
+typedef struct {
+    /* 0x00 */ u8 unk_00[0x84];
```

Comment:
sorry I keep forgetting to mention this, but I like to differentiate intentional u8 arrays from unknown data by using `char` as the type for the unknown space
```suggestion
    /* 0x00 */ char unk_00[0x84];
```

Reply:
Misunderstood your last suggestion on this, thought you meant char arrays when we figured it was string-related and u8 otherwise